fw/apm.c

```diff
@@ -440,15 +440,17 @@ typedef struct {
 /*
  * Accounting entry for particular HTTP code.
  *
- * @history	- ring buffer of history records ;
+ * @history	- ring buffer of history records;
  * @hmcfg	- pointer to structure with settings for particular HTTP code;
  * @rsum	- current amount of responses from @history ring buffer;
+ * @total	- total amount of responses during all the time;
```

Comment:
Does this include multiple reconfiguration?

Reply:
No, reconfiguration isn't allowed for these directives (server_failover_http and health_stat_server).